src/autocomplete.js

```diff
@@ -235,9 +242,30 @@ class Autocomplete extends Component {
   }
 
   render() {
-    // eslint-disable-next-line no-unused-vars
-    const {ref, onChange, ...rest} = this.props
-    return <div {...rest} ref={this.ref} />
+    const {
+      children,
+      // eslint-disable-next-line no-unused-vars
+      defaultHighlightedIndex,
+      getA11yStatusMessage,
+      // eslint-disable-next-line no-unused-vars
+      onChange,
+      // eslint-disable-next-line no-unused-vars
+      ref,
```

Comment:
Can get rid of this reference here too because we can't get access to `this.props.ref` if we wanted to. A real shame.